Algorithms/Graphs/DijkstraShortestPaths.cs

```diff
@@ -1,308 +1,167 @@
+                    var delta = _distances[currentVertexIndex] != Infinity ? _distances[currentVertexIndex] + outgoingEdge.Weight : Infinity;
 
-                    // Relax the edge
-                    // if check is true, a shorter path is found from current to adjacent
                     if (delta < _distances[adjacentIndex])
                     {
-                        _edgeTo[adjacentIndex] = edge;
                         _distances[adjacentIndex] = delta;
-                        _predecessors[adjacentIndex] = currentIndex;
+                        _predecessors[adjacentIndex] = currentVertexIndex;
 
-                        // decrease priority with a new distance if it exists; otherwise enqueque it
-                        if (minPQ.Contains(edge.Destination))
+                        if (_minPriorityQueue.Contains(outgoingEdge.Destination))
                         {
-                            minPQ.UpdatePriority(edge.Destination, delta);
+                            _minPriorityQueue.UpdatePriority(outgoingEdge.Destination, delta);
                         }
                         else
                         {
-                            minPQ.Enqueue(edge.Destination, delta);
+                            _minPriorityQueue.Enqueue(outgoingEdge.Destination, delta);
                         }
                     }
-                }//end-foreach
-            }//end-while
-        }
-
-        /// <summary>
-        /// Constructors helper function. Initializes some of the data memebers.
-        /// </summary>
-        private void _initializeDataMembers(TGraph Graph)
-        {
-            _edgesCount = Graph.EdgesCount;
-            _verticesCount = Graph.VerticesCount;
-
-            _distances = new long[_verticesCount];
-            _predecessors = new int[_verticesCount];
-            _edgeTo = new WeightedEdge<TVertex>[_edgesCount];
-
-            _nodesToIndices = new Dictionary<TVertex, int>();
-            _indicesToNodes = new Dictionary<int, TVertex>();
-
-            // Reset the information arrays
-            var i = 0;
-            foreach (var node in Graph.Vertices)
-            {
-                if (i >= _verticesCount)
-                {
-                    break;
                 }
-
-                _edgeTo[i] = null;
-                _distances[i] = Infinity;
-                _predecessors[i] = NilPredecessor;
-
-                _nodesToIndices.Add(node, i);
-                _indicesToNodes.Add(i, node);
-
-                ++i;
             }
         }
 
-        /// <summary>
-        /// Constructors helper function. Checks Optimality Conditions:
-        /// (i) for all edges e:            distTo[e.to()] <= distTo[e.from()] + e.weight()
-        /// (ii) for all edge e on the SPT: distTo[e.to()] == distTo[e.from()] + e.weight()
-        /// </summary>
-        private bool _checkOptimalityConditions(TGraph graph, TVertex source)
+        private void Initialize()
```

Comment:
Please use `thisCase` for private methods and `ThatCase` for public methods.

Reply:
Thanks for suggestion, fixed private naming method to be consistent with other classes.